cluster/node-pools/master-default/userdata.yaml

```diff
@@ -427,6 +430,31 @@ write_files:
             requests:
               cpu: 25m
               memory: 25Mi
+        {{- if index .Values "EtcdEncryptionKey" }}
```

Comment:
This is a guard to work with older versions of CLM that doesn't pass Stack Outputs in the values.